app/models/service_template.rb

```diff
@@ -234,12 +234,20 @@ def set_ownership(service, user)
     service.save
   end
 
-  def self.default_provisioning_entry_point
-    '/Service/Provisioning/StateMachines/ServiceProvision_Template/default'
+  def self.default_provisioning_entry_point(service_type = 'atomic', _prov_type = nil)
+    if service_type == 'atomic'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogItemInitialization'
+    elsif service_type == 'composite'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogBundleInitialization'
+    end
+  end
```

Comment:
If we drop the use of `service_type` this could becomes:

```ruby
def self.default_provisioning_entry_point(prov_type)
  case prov_type
  when nil            # Catalog Bundle
    '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogBundleInitialization'
  when 'generic'
    '/Service/Provisioning/StateMachines/ServiceProvision_Template/default'
  else
    '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogItemInitialization'
  end
end
```

@tinaafitz For generic services would you recommend using `CatalogItemInitialization`?  The other option besides `default` is `clone_to_service` which performs the `GroupSequenceCheck` step which may be better to use here.  Thoughts?